Sources/PayPalDataCollector/PayPalDataCollector.swift

```diff
@@ -1,44 +1,37 @@
 import Foundation
 import PPRiskMagnes
-#if canImport(PaymentsCore)
-import PaymentsCore
-#endif
 
 /// Enables you to collect data about a customer's device and correlate it with a session identifier on your server.
 public class PayPalDataCollector {
 
     // MARK: - Properties
 
-    private let config: CoreConfig
     private let magnesSDK: MagnesSDKProtocol
     private let deviceInspector: DeviceInspectorProtocol
+    private let magnesEnvironment: MagnesSDK.Environment
 
     // MARK: - Initializers
 
     /// Construct an instance to collect device data to send to your server.
-    /// - Parameter config: configuration to use when collecting device data
-    public convenience init(config: CoreConfig) {
-        self.init(config: config, magnesSDK: MagnesSDK.shared(), deviceInspector: DeviceInspector())
+    /// - Parameter environment: enviroment for the data collector
+    public convenience init(environment: PayPalDataCollectorEnvironment) {
+        self.init(environment: environment, magnesSDK: MagnesSDK.shared(), deviceInspector: DeviceInspector())
     }
 
     /// internal constructor for testing
-    init(config: CoreConfig, magnesSDK: MagnesSDKProtocol, deviceInspector: DeviceInspectorProtocol) {
-        self.config = config
+    init(environment: PayPalDataCollectorEnvironment, magnesSDK: MagnesSDKProtocol, deviceInspector: DeviceInspectorProtocol) {
+        magnesEnvironment = {
+            switch environment {
+            case .sandbox:
+                return .SANDBOX
+            case .production:
+                return .LIVE
+            }
+        }()
```

Comment:
I think we can clean this up a bit by moving this logic to the PayPalDataCollectorEnvironment enum. Then in there you could do:
```
var magnesEnvironment: MagnesSDK.Environment {
    switch self {
    case .sandbox:
        return .SANDBOX
    case .production:
        return .LIVE
    }
}
```
And in the init here just do: `magnesEnvironment = environment.magnesEnvironment`